mmpose/models/detectors/pose_lifter.py

```diff
@@ -156,6 +243,13 @@ def forward_dummy(self, input):
         if self.with_keypoint:
             output = self.keypoint_head(features)
 
+        if self.with_traj:
```

Comment:
Please modify L240, L242, and L244: features -> output, to ensure that the variable "output" exists.